include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -146,6 +166,16 @@ class CMAES
 
   //! The selection policy used to calculate the objective.
   SelectionPolicyType selectionPolicy;
+
+  //! Step size damping.
+  double ds;
```

Comment:
Just for consistency, it might be beter to name this `stepSizeDamping` since the method to access it is `StepSizeDamping()`.  (Similar for `CumulationConstant()`.)  If we are exposing these parameters, should we also add constructor parameters for these also?  I think maybe I don't fully follow what these parameters are for (or how a user would set them).

Reply:
I'll change the name for sure. I included those two based on Marcus' suggestion in the issue discussion. These parameters are used in the update of the variables. It provides some extra flexibility for the user, but I wonder if including them in the constructor would become more confusing to the user, or how useful they would be (I use CMA-ES a lot for my work, and have never really toyed with these values). Maybe @zoq can shed some light on this?

Reply:
Ah, that's fair.  It doesn't matter to me if the variables are included or not, but if we do include them then I think we should make them accessible from the constructor. :+1: